docs/multicluster/user-guide.md

```diff
@@ -81,6 +81,12 @@ antrea-agent.conf: |
     namespace: "" # Change to the Namespace where antrea-mc-controller is deployed.
 ```
 
+In order for Multi-cluster features to work, `enableGateway` should be set to true by the user,
```

Comment:
I was reading this again, and I think a small improvment could be:

> In order for Multi-cluster features to work,  it is necessary for `enableGateway` to be set to true by the user,

Reply:
Done, thanks.